tutorials/colors/colormap-manipulation.py

```diff
@@ -255,6 +255,29 @@ def plot_linearmap(cdict):
 
 plot_examples([cmap1, cmap2])
 
+#############################################################################
+# .. _reversing-colormap:
+#
+# Reversing a colormap
+# ====================
+#
+# Any colormap that is a registered as a matplotlib `.Colormap` can be reversed
+# using the ``reversed`` method of the colormap object. For example:
+
+colors = ["#ffffcc", "#a1dab4", "#41b6c4", "#2c7fb8", "#253494"]
+my_cmap = ListedColormap(colors, name="my_cmap")
+# register
+mpl.colormaps.register(cmap=my_cmap)
+# reverse custom colormap
+mpl.colormaps.register(cmap=my_cmap.reversed())
+
+plot_examples(['my_cmap', 'my_cmap_r'])
```

Comment:
You're mixing reversing and registering here, which are completely independent concepts.

```suggestion
# One can created a reversed version of any colormap object by calling
# `.Colormap.reversed`

colors = ["#ffffcc", "#a1dab4", "#41b6c4", "#2c7fb8", "#253494"]
my_cmap = ListedColormap(colors, name="my_cmap")

my_cmap_r = my_cmap.reversed()

plot_examples(['my_cmap', 'my_cmap_r'])
```

Optionally, you may add a separate section on registering colormaps if you want.

Reply:
Oh sorry, I haven't read your example carefully. What you should do here is pass the objects to `plot_examples()` not the names. `plot_examples()` can handle both, but similar to the other exmaples above, this directly works on the objects.